### ISSUE TITLE
[03-Gaussians] Unclear what gaussian point value means

### ISSUE THREAD

**Issue by pikeas** [rlabbe/Kalman-and-Bayesian-Filters-in-Python#294]
The description of _cumulative probability distribution_ is clear, however the adjacent description of the _gaussian_ is not.

> What is the probability of a the temperature being exactly 22°C? Intuitively, 0. These are real numbers, and the odds of 22°C vs, say, 22.00000000000017°C is infinitesimal. Mathematically, what would we get if we integrate from 22 to 22? Zero.

> We can think of this in Bayesian terms or frequentist terms. As a Bayesian, if the thermometer reads exactly 22°C, then our belief is described by the curve - our belief that the actual (system) temperature is near 22°C is very high, and our belief that the actual temperature is near 18 is very low.

It sounds like `gaussian(x, mu, var)` should only be used to quantify relative belief:

    print(gaussian(x=[3.0, 2.0], mean=2.0, var=1))
    [0.37754067 0.62245933]

In this example from the chapter, the true value is 1.65x as likely to be "near" 2 as it is to be "near" 3.

    print(gaussian(x=3.0, mean=2.0, var=1))
    0.24197072451914337

This seems to mean that we are 24% sure the value is "near" 3. The docs for `gaussian()` say:

    x : scalar or array-like
      The value for which we compute the probability

There's a conceptual leap here from ```the probability the temperature is between 21 and 23``` to ```the probability the temperature is 22``` which is hard to follow, after previously establishing that the probability at any specific value is 0.

Love the book overall!

**Comment by dimidagd** [comment]
In many cases PDF can be misunderstood for PMF, https://en.wikipedia.org/wiki/Probability_mass_function

**Comment by rlabbe** [comment]
I've fixed the docstring in filterpy and pushed it, but have not yet created a new release of it. 

gaussian() does not compute the probability, but the pdf. `gaussian(3,2,1)` computes the pdf of N(2,1) at the location 3. This is NOT a probability, but the density at that point. 

In practice you would not call `gaussian` like that, but instead like

    gaussian(np.linspace(0, 4, 1000), 2, 1)

to get the pdf between 0 and 4.
